terragrunt/aws/config/wafv2_logs.tf

```diff
@@ -0,0 +1,243 @@
+#
+# Config Rule
+#
+resource "aws_config_config_rule" "cbs_wafv2_logs_rule" {
+  name        = "cbs_wafv2_logs_rule"
+  description = "Checks that the WAFV2 ACL's are logging to either S3 or Kinesis."
+
+  source {
+    owner             = "CUSTOM_LAMBDA"
+    source_identifier = aws_lambda_function.cbs_wafv2_logs_rule.arn
+
+    source_detail {
+      message_type                = "ScheduledNotification"
+      maximum_execution_frequency = var.config_max_execution_frequency
+    }
+  }
+
+  scope {
+    compliance_resource_types = ["AWS::WAFv2::WebACL"]
+  }
+
+  depends_on = [
+    aws_lambda_permission.cbs_wafv2_logs_rule,
+  ]
+}
+
+#
+# Lambda function used by the ConfigRule
+#
+data "archive_file" "cbs_wafv2_logs_rule" {
+  type        = "zip"
+  source_file = "config_rules/wafv2_logs/webacl_logs_rule.py"
+  output_path = "/tmp/cbs_wafv2_logs_rule.zip"
+}
+
+resource "aws_lambda_function" "cbs_wafv2_logs_rule" {
+  filename      = "/tmp/cbs_wafv2_logs_rule.zip"
+  function_name = "cbs_wafv2_logs_rule"
```

Comment:
Can you change this to either have a `Cbs` or `cbs-` prefix (for the AWS nuke config exemptions).